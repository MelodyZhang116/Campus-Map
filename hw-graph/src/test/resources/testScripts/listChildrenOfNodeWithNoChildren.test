# test whether the children of a node(no children) can be listed correctly.

CreateGraph A
AddNode A n1
ListChildren A n1
