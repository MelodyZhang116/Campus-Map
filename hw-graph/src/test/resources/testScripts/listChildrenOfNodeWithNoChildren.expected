# test whether the children of a node(no children) can be listed correctly.

created graph A
added node n1 to A
the children of n1 in A are:
